recipes/libev/build.sh

```diff
@@ -0,0 +1,3 @@
+./configure --prefix="${PREFIX}"
+make
+make install
```

Comment:
Oh weird. Alright. I looked at the makefile and thought there was a target for check. 🤷‍♂️